### PR TITLE
Remote work 2

### PR DESCRIPTION
**Pull request by Lydias303**
Part 2 of how to work remotely effectively.

### CODE REVIEW

File: _posts/2016-01-14-remote-working-part-two.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+tags: development
+author_github:
+- Lydias303
+date: 2016-01-14 10:31:37
+comments: true
+banner_image: home-work.jpg
+---
+###How to work effectively remotely:: Part II Community and Culture
+
+I began this blog post after attending a great talk at [Denver Startup Week](http://www.denverstartupweek.org/) where four leaders in the remote industry shared helpful tips and answered questions from the audience about working remotely. We left off in Part One of this blog discussing solutions to some challenges of environment and productivity when working remotely. We will continue our discussion with some ways to establish good communication and positive culture remotely. Creating a company culture is a challenging prospect regardless of the teams distribution. The positive that occurs when fostering culture in a remote setting is that it is never assumed that culture will occur magically. People enter the situation with the knowledge that building a remote culture is challenging and are aware of the importance of cultivating it within their team.
```
Comment (laurawhalin) [inline _posts/2016-01-14-remote-working-part-two.md]: missing apostrophe in "team's distribution"

File: _posts/2016-01-14-remote-working-part-two.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+###Culture
+__Find your tools!__ Find tools you use on a daily basis that aid in communication and establish your virtual community.  [Slack](https://slack.com/) is our jumping of point when it comes to communication and culture. Its where work discussions happen but also where inside jokes, witty banter and giphys roam. It is important to stay involved. Don't think of yourself as being on an island, there is always someone to chat with and get feedback from. __Pair with your team mates!__ [Screenhero](https://screenhero.com/), [Google Hangouts](https://hangouts.google.com/), [Speak](https://speak.io/) are just a few examples of ways you can pair remotely.
```
Comment (laurawhalin) [inline _posts/2016-01-14-remote-working-part-two.md]: change to "jumping off point" (missing extra f in off)
add apostrophe to "It's where work discussions"

File: _posts/2016-01-14-remote-working-part-two.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+While most of the culture shaping occurs from afar it is so important to get together. Use company retreats to help foster culture through team building, dinners and other group events. These events help you learn about each other and grow closer as a team. 
+
+<img style="margin:auto;" src="{{ site.url }}/assets/images/culture.jpg">
+
+Allow your team opportunities to get involved and give back to your community. Go to meetups, talks, conferences, __get excited about what your doing.__ Donate your time to non profits, volunteer to help people learn to code. Being involved in the community allows you to grow as a developer and in turn become a better co worker. On our team Fridays are our 20 percent time. We can use this time to self study, mentor, volunteer, blog ect. It has been a great way to improve and explore together and individually.
```
Comment (laurawhalin) [inline _posts/2016-01-14-remote-working-part-two.md]: change your to you're in "get excited about what you're doing"

File: _posts/2016-01-14-remote-working-part-two.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+<img style="margin:auto;" src="{{ site.url }}/assets/images/culture.jpg">
+
+Allow your team opportunities to get involved and give back to your community. Go to meetups, talks, conferences, __get excited about what your doing.__ Donate your time to non profits, volunteer to help people learn to code. Being involved in the community allows you to grow as a developer and in turn become a better co worker. On our team Fridays are our 20 percent time. We can use this time to self study, mentor, volunteer, blog ect. It has been a great way to improve and explore together and individually. 
+
+My favorite thing about remote culture is __trust__. When working remotely you have to trust your team mates. We aren't bound by office rules, hours or schedules. We have freedom to work when we like and our team mates have to trust in us to deliver. We want to do a good job and we accomplish that with what we deliver at the end of every week. __Our trust in each other is the platform the rest of our culture built upon.__
```
Comment (laurawhalin) [inline _posts/2016-01-14-remote-working-part-two.md]: "teammates" is one word

File: _posts/2016-01-14-remote-working-part-two.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+When discussing culture in co-located and remote teams concerns about sub cultures were discussed. While they are inevitable it is important to remind co-located and remote teams alike to be inclusive and focus on maintaining the __"big culture"__. One speaker encouraged people to _"Think of your company as a human being. It has an opinion and characteristics."_ When subcultures begin to branch out work harder to make people feel like they are a part of something higher. _"Also spend some money. Get them together."_ Break up the subculture by focusing on the company as a whole. Try taking the work element out and do stuff remotely together. Like playing a video game online. __Stream the same music__ on everyones computer, allow team members to add their music to a que. Listing to the same music is a great and simple way to share something throughout the day. Think outside the box when it comes to maintaining the big culture.
```
Comment (laurawhalin) [inline _posts/2016-01-14-remote-working-part-two.md]: fix capitalization in "Also spend some money" (ALso is how it is now)
you have both "sub culture" and "subculture" in this paragraph. I'd stick with "subculture"
add apostrophe to "everyone's "
que should be "queue"

File: _posts/2016-01-14-remote-working-part-two.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+###Communication
+__Use your tools!__ Determine a primary form of communication that can provide your team with streamline and open communication. [Slack](https://slack.com/) as I mentioned before allows for quick and easy communication and is the remote workers virtual office. Informal chatting in group channels allows for inclusion regardless of location and keeps team members involved.
```
Comment (laurawhalin) [inline _posts/2016-01-14-remote-working-part-two.md]: I think you're missing a "d" at the end of streamline. 
add apostrophe to "remote worker's virtual office"

File: _posts/2016-01-14-remote-working-part-two.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+###Communication
+__Use your tools!__ Determine a primary form of communication that can provide your team with streamline and open communication. [Slack](https://slack.com/) as I mentioned before allows for quick and easy communication and is the remote workers virtual office. Informal chatting in group channels allows for inclusion regardless of location and keeps team members involved. 
+
+While written communication is pivotal, video conferencing is an invaluable tool for remote teams. For teams with a co-located office group calls include remote workers in company wide meetings and keep the team in sync. Smaller team video calls are of equal importance. Discussions such as this work best when a team member is selected to drive the meeting and a record of feedback is produced. The real time record prevents missed information and is a useful reference in the future.
```
Comment (laurawhalin) [inline _posts/2016-01-14-remote-working-part-two.md]: 2nd sentence is a little hard to understand. Maybe add a comma after the word office?

File: _posts/2016-01-14-remote-working-part-two.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+<img style="margin:auto;" src="{{ site.url }}/assets/images/communication.png">
+
+A __daily stand up__ hangout is a great way to keep everyone in sync and has been extremely helpful for our team. It ensures the team is informed and communicating on a daily basis. Use this time to ask questions, open discussions and ensure you are up to speed with the rest of your team. It is important to __have a presence__ when working remotely. One great way to accomplish this is to put a face to your name. While Im sure most remote workers can admit their guilt of shutting off their camera on a hangout due to lack of sleep, make up or pants. It is important to show your face during meetings. It allows the speaker to be more aware of their audience allows you to have a presence from hundreds of miles away. __Weekly Retro's__ are a great tool as well. Use this time to compliment hard work, vent about what you struggled with, basically use it as a team therapy session.
```
Comment (laurawhalin) [inline _posts/2016-01-14-remote-working-part-two.md]: add apostrophe in "I'm"
add the word "and" between "audience allows"
remove apostrophe from "Retros"

File: _posts/2016-01-14-remote-working-part-two.md [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+
+Screen sharing enables us to pair remotely and seek help when stuck on a problem. When you reach out to for help in a case like this make sure you are prepared. Ensure you understand your problem and can adequately explain and show your dilemma. Give the person helping you the tools they need to guide you to a solution.
```
Comment (laurawhalin) [inline _posts/2016-01-14-remote-working-part-two.md]: remove the word "to" in "reach out to for help"

**Comment by laurawhalin** [comment]
Great info in here @Lydias303 !!!

**Comment by Lydias303** [comment]
Dang typo city! Thank you @laurawhalin

**Pull request merged**